### DISCUSSION TITLE
Set cache size

### DISCUSSION THREAD

**Discussion by evwltrs** [kubilus1/autoortho#81] [Q&A]
How can I specify a maximum size for the cache?

**Comment by Simssmith** [comment]
I am interested in this as well.  How/where in the app is cache lifecycle configuration maintained?

**Comment by jonaseberle** [comment]
There is no maximum size for the cache implemented (yet).

**Comment by BinaryOutlook** [comment]
Believe this was a function, but was removed.

**Reply by jonaseberle** [reply]
Since very early it managed the used RAM for tiles that are currently being accessed. 

But there has not been any cache pruning for the disk cache.

**Reply by Glasairmell** [reply]
Just curious.  Why would you want to do that?  Would you want to keep the BI files you have downloaded and not have to re download them?
